core/config/docs/core.toml

```diff
@@ -509,6 +509,26 @@ DeltaReconcile = '1m' # Default
 # but the host and port must be fully specified and cannot be empty. You can specify `0.0.0.0` (IPv4) or `::` (IPv6) to listen on all interfaces, but that is not recommended.
 ListenAddresses = ['1.2.3.4:9999', '[a52d:0:a88:1274::abcd]:1337'] # Example
 
+[Capabilities.GatewayConnectorConfig]
+# ChainIDForNodeKey is the ChainID of the network
+ChainIDForNodeKey = '0xfd29Dd9C980D715a64dace97F7A2AB98bcaE0fed' # Default
+# NodeAddress is Workflow Node address
+NodeAddress = '0x68902d681c28119f9b2531473a417088bf008e59' # Example
+# DonID is the Id of the Don
+DonID = 'example_don' # Example
+# WsHandshakeTimeoutMillis is Websocket handshake timeout
+WsHandshakeTimeoutMillis = 1000 # Example
+# AuthMinChallengeLen is the minimum number of bytes in Authentication
+AuthMinChallengeLen = 10 # Example
+# AuthTimestampToleranceSec is Authentication timestamp tolerance
+AuthTimestampToleranceSec = 10 # Example
+
+[[Capabilities.GatewayConnectorConfig.Gateways]]
```

Comment:
```suggestion
[Capabilities.GatewayConnector]
# ChainIDForNodeKey is the ChainID of the network
ChainIDForNodeKey = '0xfd29Dd9C980D715a64dace97F7A2AB98bcaE0fed' # Default
# NodeAddress is Workflow Node address
NodeAddress = '0x68902d681c28119f9b2531473a417088bf008e59' # Example
# DonID is the Id of the Don
DonID = 'example_don' # Example
# WsHandshakeTimeoutMillis is Websocket handshake timeout
WsHandshakeTimeoutMillis = 1000 # Example
# AuthMinChallengeLen is the minimum number of bytes in Authentication
AuthMinChallengeLen = 10 # Example
# AuthTimestampToleranceSec is Authentication timestamp tolerance
AuthTimestampToleranceSec = 10 # Example

[[Capabilities.GatewayConnector.Gateways]]
```

Reply:
Actually, does `Connector` add value? Or would this make sense and be clear to users as just `Gateway`?

Reply:
Yeah Connector is important as it distinguishes when a node connects to gateways or is a gateway itself. Similarly we talk about Gateway handlers and Connector handlers. Let's keep it.